eppo/src/test/java/cloud/eppo/android/FlagConfigResponseDeserializerTest.java

```diff
@@ -0,0 +1,143 @@
+package cloud.eppo.android;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import cloud.eppo.android.dto.Allocation;
+import cloud.eppo.android.dto.FlagConfig;
+import cloud.eppo.android.dto.FlagConfigResponse;
+import cloud.eppo.android.dto.OperatorType;
+import cloud.eppo.android.dto.Range;
+import cloud.eppo.android.dto.Shard;
+import cloud.eppo.android.dto.Split;
+import cloud.eppo.android.dto.TargetingCondition;
+import cloud.eppo.android.dto.Variation;
+import cloud.eppo.android.dto.VariationType;
+import cloud.eppo.android.dto.adapters.FlagConfigResponseAdapter;
+
+public class FlagConfigResponseDeserializerTest {
+
+    private final Gson gson = new GsonBuilder()
+            .registerTypeAdapter(FlagConfigResponse.class, new FlagConfigResponseAdapter())
+            .create();
+
+    @Test
+    public void testDeserializePlainText() throws IOException {
+        File testRac = new File("src/androidTest/assets/flags-v1.json");
```

Comment:
This is coming from https://github.com/Eppo-exp/sdk-test-data/blob/main/ufc/flags-v1.json right?

So this would actually be

```suggestion
        File testUfc = new File("src/androidTest/assets/flags-v1.json");
```

?

Reply:
Yes! I should rename the local variable